ivy_tests/test_ivy/test_frontends/test_torch/test_nn/test_functional/test_pooling_functions.py

```diff
@@ -271,9 +256,6 @@ def test_torch_max_pool2d(
     on_device,
 ):
     dtype, x, kernel, stride, pad, dilation = x_k_s_p
-    # Torch ground truth func expects input to be consistent
-    # with a channels first format i.e. NCHW
-    x[0] = x[0].reshape((x[0].shape[0], x[0].shape[-1], *x[0].shape[1:-1]))
     pad = (pad[0][0], pad[1][0])
```

Comment:
the returned padding is a list for each spatial dim like in 1d case it will be a list with a tuple of two elements and the same with 2d and so on but torch pooling functions only accept an integer or a tuple of 1 element in their padding attr and a tuple of two elements in the 2d case